pyproject.toml

```diff
@@ -206,7 +206,7 @@ source = [
     "tests",
     "fastapi"
 ]
-context = '${CONTEXT}'
```

Comment:
Ah, not sure what this piece was for TBH 😅. Decided to remove it though.

Reply:
This env var is used by/with GitHub Actions so that coverage can know where to store the coverage files, that way they can be combined later. So I think doing this trick wouldn't work. :thinking: